__tests__/app.test.js

```diff
@@ -132,20 +132,88 @@ describe("GET - 200: /api/reviews/:review_id/comments", () => {
         expect(body.comments).toEqual([]);
       });
   });
-    test("GET - 404: responds with an error message, when passed an invalid review_id", () => {
-      return request(app)
-        .get("/api/reviews/999/comments")
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Comments with review id 999 not found!");
-        });
-    });
-    test("GET - 400: responds with an error message, when passed an invalid request", () => {
-      return request(app)
-        .get("/api/reviews/nonsense/comments")
-        .expect(400)
-        .then(({ body }) => {
-          expect(body.msg).toBe("invalid query!");
+  test("GET - 404: responds with an error message, when passed an invalid review_id", () => {
+    return request(app)
+      .get("/api/reviews/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Comments with review id 999 not found!");
+      });
+  });
+  test("GET - 400: responds with an error message, when passed an invalid request", () => {
+    return request(app)
+      .get("/api/reviews/nonsense/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid query!");
+      });
+  });
+});
+
+describe.only("/api/reviews/:review_id/comments", () => {
```

Comment:
remove .only when doing Pull Requests

Extra tests required;

app ignores unnecessary req body keys

is there still a test for invalid username?